examples/core/source_thermic_example_tmp.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
```

Comment:
please consider refactor this into unittest or example. 
if needed, add the speos file used into the tests/assert folder